scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/SchedulerAppDriver.java

```diff
@@ -35,11 +36,12 @@ public static void main(final String[] args) {
 
     //create objects in each service abstraction layer (mirroring MerlinApp)
     final var merlinService = new GraphQLMerlinService(config.merlinGraphqlURI());
```

Comment:
Is this `scheduler` of a new service? Should we not use the `merlin` in `GraphQLMerlinService`

Reply:
Maybe `GraphQLSchedulerService`

Reply:
Thanks for looking through this! I think I see where there may be some confusion.

So the line below it is the actual schedule service (`specificationService`, where `specification` is referring to a scheduler spec.). The `GraphQLMerlinService` is a GraphQL service that queries Merlin's GraphQL API.

Reply:
Currently `GraphQLMerlinService` is used to query for plans from Merlin. Kenny & Adrien will be working on removing this `GraphQLMerlinService` altogether and relying on `SpecificationService` to handle Merlin queries internally.

Reply:
Thanks for the clarification guys.